File: Numbers/Floor Cost/floor.py
# Floor Cost
# Calculate the total cost of tile it would take to cover a floor plan of width and height, using a cost entered by the user

width = float(input("How wide is the floor (ft)? "))
height = float(input("How long is the floor (ft)? "))
unit_price = float(input("How much per square foot? "))

area = width * height
total = unit_price * area

print("The total cost of the floor is ${:.2f}".format(total))

File: Numbers/Floor Cost/floor.cpp
// Floor Cost
// Calculate the total cost of tile it would take to cover a floor plan of width and height, using a cost entered by the user

#include <iostream>
using std::cout; using std::cin; using std::endl;

int main() {
	double width, length, area, unit_cost, total_cost;

	cout << "How wide is the floor (ft)? ";
	cin >> width;
	cout << "How long is the flood (ft)? ";
	cin >> length;
	cout << "How much per square foot? ";
	cin >> unit_cost;

	area = width * length;
	total_cost = area * unit_cost;

	cout << "The total cost of the floor is $" << total_cost << endl;
}